infrastructure/Taskfile.yml

```diff
@@ -681,7 +681,11 @@ tasks:
             --branches "^({{ .BRANCHES }})$" \
             --project {{.PROJECT_NAME}}
         - task: lagoon:project:ensure:github-registry-credentials
+          vars:
+            PROJECT_NAME: "{{.PROJECT_NAME}}"
         - task: lagoon:project:deploykey
+          vars:
+            PROJECT_NAME: "{{.PROJECT_NAME}}"
```

Comment:
er det med vilje at det samme bliver gjort to gange?

Reply:
Hvor? Hvad?